FleekHQ/minio
# Make handlers in sync to CRUD operations

## Description
Currently these handlers are happening asynchronously after the crud operation and to avoid concurrency issues we need to put them under the same bucket level lock that the s3 ledger store is doing.

## Motivation and Context
Async operations for this causes data to be lost.

## How to test this PR?
Create bucket or file and it should call lambda to do CRUD handling in-sync (need to check threads db or dns entry after to confirm)

## Types of changes
-  Bug fix (non-breaking change which fixes an issue)

## Checklist:
